### PR TITLE
PE-21 | Backend fully conntected. Springboot connected with MongoDB.

### PR DESCRIPTION
**Pull request by davidsquines**
What I did:

- Created basis for our backend.

- Connected Springboot Application with MongoDB.

- Created "User" entity with "firstname" and "lastname" values only.

How to do:

- Download Mongo Compass UI. This will be needed to visualize the data table. 

- Clone the "project-eureka-spring" repository into your machine

- After compass has been downloaded, enter the correct connection string for our database. Contact Tony or me for the connection string. 

- Checkout and pull from the branch "PE-21-SetUpMongo.

- Navigate to the **src/main/resources/application.properties**. Inside you should see a single url starting with spring.data.mongo.uri. You will need to change the fields "<user>, <password>, and <Auth_DB>" to the correct values. **Ensure that when you replace these values, delete the "<" and ">" from the final input fields**. Contact Tony N. or I for these fields. 
- Navigate to the file "ProjectEurekaBackendApplication.java" and find the following area Screen Shot 2020-12-02 at 1 32 44 PM
- Change the values for the first and last name of the new users to be inserted into the database. 

- Run the application by pressing the "play" on Visual Studio Code  or "run" button on your IDE Screen Shot 2020-12-02 at 1 34 22 PM
- The new users should be inputed into the database. Verify via the Mongo Compass app by navigating to the collection "Users" in the "Project-Eureka" database. Your compass should now contain the new users you inputed.  Screen Shot 2020-12-02 at 1 43 28 PM

### CODE REVIEW

**Comment by MaxLian11** [comment]
Works totally fine on my end. The only thing I would change is to keep Models, Repositories and Services in separate folders, as we are going to have more tables in the future. Controller folder can also be created for future development

**Review by MaxLian11 — CHANGES_REQUESTED** [review]
Please read the comment and create new folders

**Review by dominikyz — APPROVED** [review]
Code looks good to me. As Max suggested separate into different folders. Great job!

File: src/main/java/com/ProjectEureka/backend/User.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package com.ProjectEureka.backend;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
```
Comment (tonynguyen98) [inline src/main/java/com/ProjectEureka/backend/User.java]: Delete these.

**Review by tonynguyen98 — CHANGES_REQUESTED** [review]
It's working on my end! I also agree on the two other reviews, please move these files into proper directories, using lowercase formatting for folder names.

**Review by MaxLian11 — APPROVED** [review]
Looks good now

**Review by tonynguyen98 — APPROVED** [review]
Good work! It's working as expected.

**Pull request merged**